src/main/java/fr/esgi/al5_2/Tayarim/entities/Administrateur.java

```diff
@@ -0,0 +1,24 @@
+package fr.esgi.al5_2.Tayarim.entities;
+
+import jakarta.persistence.*;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.AvoidStarImportCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Using the '.*' form of import should be avoided - jakarta.persistence.*.